nemo/collections/tts/models/fastspeech2.py

```diff
@@ -75,7 +75,12 @@ def __init__(self, cfg: DictConfig, trainer: 'Trainer' = None):
 
         # Parser and mappings are used for inference only.
         self.parser = parsers.make_parser(name='en')
-        with open(cfg.mappings_filepath, 'r') as f:
+        if 'mappings_filepath' in cfg:
+            mappings_filepath = cfg.get('mappings_filepath')
+        else:
+            mappings_filepath = 'mappings.json'
+        mappings_filepath = self.register_artifact('mappings_filepath', mappings_filepath)
+        with open(mappings_filepath, 'r') as f:
```

Comment:
Extract this step into another function and use the ptl decorator `rank_zero_only` so that during DDP only zero rank writes it (file corruption otherwie).